### DISCUSSION TITLE
Ctrl+F shows 0/0 on some directory

### DISCUSSION THREAD

**Discussion by etrnal70** [PatrickF1/fzf.fish#94] [Q&A] [answered]
On some directories, using ctrl + F shows empty result e.g. on home. Strangely the other command like show git files is working, only ctrl+f isn't. Also, using ctrl+f on some other directory is working.

**Comment by etrnal70** [comment]
Closing this since it seems the culprit is from my borked git directory

**Comment by PatrickF1** [comment]
Hi @etrnal70, could you tell me more about what the problem was so I can possibly include it in the troubleshooting section?

**Comment by etrnal70** [comment, marked as answer]
I think it's because i initialize my dotfiles repo in home. Since i put * in .gitignore, based on the default Ctrl+F command, it should avoid any subdirectories from home that's not added to my dotfiles repo. I can confirmed it because when i try the command in another local git repo, it works fine
